src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php

```diff
@@ -78,6 +78,10 @@ public function create(string $resourceClass): ResourceMetadata
 
             $resourceMetadata = $resourceMetadata->withItemOperations($this->createOperations($methods));
         } else {
+            if (!$itemOperations) {
+                $itemOperations = ['placeholder' => ['method' => 'GET', 'read' => false, 'serialize' => false, 'status' => 404]];
```

Comment:
The operation name should be `get` instead of `placeholder`.

Reply:
Can't we use `'output' => false` here?

Reply:
I wanted to be able to differentiate the get operation but sure can do!

Reply:
> to be able to differentiate the get operation

I know. I'm against that. :laughing:

I think there should be nothing special about it. Just a good / safe default when the `get` item operation is missing.